grpc/src/test/java/com/linecorp/armeria/internal/server/grpc/GrpcDocServicePluginTest.java

```diff
@@ -103,6 +107,18 @@ void servicesTest() throws Exception {
                                                       m.name());
             });
         });
+        services.get(HttpJsonTranscodingTestServiceGrpc.SERVICE_NAME).methods().forEach(m -> {
+            m.endpoints().forEach(e -> {
+                assertThat(e.pathMapping()).isEqualTo("/armeria.grpc.testing.HttpJsonTranscodingTestService/" +
+                                                      m.name());
+            });
+        });
+        services.get(HttpJsonTranscodingTestServiceGrpc.SERVICE_NAME +
+                     GrpcDocServicePlugin.HTTP_SERVICE_SUFFIX).methods().forEach(m -> {
+            m.endpoints().forEach(e -> {
+                assertThat(e.pathMapping()).endsWith(m.examplePaths().get(0));
```

Comment:
I think this validation doesn't mean anything, but there isn't a good way for checking the path.
Having said this, it's fine as is. 😄